### PR TITLE
[rllib] Update RLlib to work with new actor scheduling behavior

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

Due to https://github.com/ray-project/ray/issues/1734, actors now can't be created in arbitrary quantities, which breaks our actor colocation code. We can add it back once support for setting cpu=None / global scheduling of those is added.

On the plus side, the scheduler now supports bin-packing and so we can remove the `force_evaluators_remote` hack.

I also fixed the YAML configs to request the proper number of resources. They were a bit off in some cases.

### CODE REVIEW

File: python/ray/rllib/optimizers/apex_optimizer.py [hunk 1]
```diff
@@ -163,12 +163,15 @@ def _init(
...
         self.learner.start()
 
-        self.replay_actors = create_colocated(
-            ReplayActor,
-            [num_replay_buffer_shards, learning_starts, buffer_size,
-             train_batch_size, prioritized_replay_alpha,
-             prioritized_replay_beta, prioritized_replay_eps, clip_rewards],
-            num_replay_buffer_shards)
+        # TODO(ekl) use create_colocated() for these actors once
+        # https://github.com/ray-project/ray/issues/1734 is fixed
```
Comment (ericl) [inline python/ray/rllib/optimizers/apex_optimizer.py]: Conveniently, the current behavior with cpu=None basically almost guarantees co-location.

Comment (richardliaw) [reply python/ray/rllib/optimizers/apex_optimizer.py]: so we should revert this change after #1734?

Comment (ericl) [reply python/ray/rllib/optimizers/apex_optimizer.py]: Maybe, it will require more testing.

**Review by richardliaw — APPROVED** [review]
approving - changes are localized and shouldn't affect much else.

**Comment by ericl** [comment]
Test failure looks unrelated.

**Pull request merged**